tests/framework/di/InstanceTest.php

```diff
@@ -46,6 +46,26 @@ public function testEnsure()
         $this->assertTrue(Instance::ensure(['class' => 'yii\db\Connection', 'dsn' => 'test'], 'yii\db\Connection', $container) instanceof Connection);
     }
 
+    /**
+     * ensure an InvalidConfigException is thrown when a component does not exist.
+     */
+    public function testEnsureNonExistingComponent()
```

Comment:
agree, but imagine you are testing in your IDE and you get an error:
* testNonExistingClassException -> "Failing test non existing class exception"

Is the developer informed completely enough? Does he instantly know the reason and source of the failing test?

Additionally. When looking at this subject from a 'test management perspective'. Is it clear enough in the 'test file' that tests already have been made for the 'ensure' method.